android/src/facebook/TiFacebookModule.java

```diff
@@ -115,6 +115,20 @@ public static void onAppCreate(TiApplication app)
 	public static TiFacebookModule getFacebookModule() {
 		return module;
 	}
+    
+    @Override
+    public void onResume(Activity activity) {
+        super.onResume(activity);
+        Log.d(TAG, "Calling activateApp");
+        AppEventsLogger.activateApp(activity);
```

Comment:
They will start receiving statistics.

The only negative consequence i can think about, is that they wont be able
to cheat Facebook on ad campaigns (getting referrals without paying for
installs)

However, in iOS version of this module, this behaviour (installs logging) is
enabled by default and cant be disabled

On Monday, 18 January 2016, Ashraf <notifications@github.com> wrote:

> In android/src/facebook/TiFacebookModule.java
> <https://github.com/appcelerator-modules/ti.facebook/pull/37#discussion_r49957188>
> :
>
> > @@ -115,6 +115,20 @@ public static void onAppCreate(TiApplication app)
> >  	public static TiFacebookModule getFacebookModule() {
> >  		return module;
> >  	}
> > +
> > +    @Override
> > +    public void onResume(Activity activity) {
> > +        super.onResume(activity);
> > +        Log.d(TAG, "Calling activateApp");
> > +        AppEventsLogger.activateApp(activity);
>
> If someone is not using AppEventsLogger, how would this affect them?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/appcelerator-modules/ti.facebook/pull/37/files#r49957188>
> .
>